### DISCUSSION TITLE
How to convert HL7 V2 messages to FHIR format using HttpListener

### DISCUSSION THREAD

**Discussion by FeiTian01** [nextgenhealthcare/connect#4654] [Q&A] [answered]
[http://www.mirthcorp.com/community/wiki/pages/viewpage.action?pageId=38798959](url)
I downloaded the channel from V2 to FHIR(channel name: Convert HL7 to FHIR Patient - Example 1)
I want to change the source of this channel to HttpListener. 
What should I do? How can I use Postman to send ADT-A01 messages

**Comment by yaseenexists** [comment, marked as answer]
Hello @FeiTian01,
Here is the channel group for the same: https://github.com/yaseenexists/example-mirth-channels/commit/8aa7a273eeb66862e254a1cf93f2225cd9f46be2
* This channel uses HTTP listener for the source.  
* File writer for destination. (I have a channel for HTTP Writer as well, but you'll need a receiving application to accept the POST request.)
* It has a preprocessor step that removes few unwanted characters from incoming HL7 file.

**Reply by MrTianFei** [reply]
Thank you so much
